geonode/geoserver/signals.py

```diff
@@ -223,19 +224,20 @@ def geoserver_post_save_local(instance, *args, **kwargs):
     instance.workspace = gs_resource.store.workspace.name
     instance.store = gs_resource.store.name
 
-    bbox = gs_resource.latlon_bbox
-
-    # FIXME(Ariel): Correct srid setting below
-    # self.srid = gs_resource.src
-
-    instance.srid_url = "http://www.spatialreference.org/ref/" + \
-        instance.srid.replace(':', '/').lower() + "/"
+    bbox = gs_resource.native_bbox
```

Comment:
maybe check if bbox is not None

Reply:
this can apply also to other parts. Can we assume native_bbox is always populated?

Reply:
by using gsconfig, yes (at least with current version).